src/SASjs.ts

```diff
@@ -42,7 +42,8 @@ const requestRetryLimit = 5;
  *
  */
 export default class SASjs {
-  private sasjsConfig = new SASjsConfig();
+  private globalSasjsConfig: SASjsConfig = new SASjsConfig();
+  private sasjsConfig: SASjsConfig = new SASjsConfig();
```

Comment:
is this an empty config?  What would happen if the`globalSasConfig` is updated, but not the `sasjsConfig` ?